llvm/tools/llvm-objcopy/ObjcopyOpts.td

```diff
@@ -93,6 +93,17 @@ defm set_section_type
          "Set the type of section <section> to the integer <type>">,
       MetaVarName<"section=type">;
 
+defm set_symbol_visibility
+    : Eq<"set-symbol-visibility",
+         "Change the visibility of a symbol to the specified type">,
+      MetaVarName<"symbol=visibility_type">;
+defm set_symbols_visibility
+    : Eq<"set-symbols-visibility",
+         "Reads a list of symbols from <filename> and changes their "
```

Comment:
Read ... change

Use an imperative sentence to match the convention.

Reply:
I copied this description from other options, localize- globalize- strip- weaken- keep- symbols all use this wording. Should I change it?